blocks/library/author/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+import AuthorBlock from './block';
+import { registerBlockType } from '../../api';
+
+registerBlockType( 'core/author', {
+	title: __( 'Author' ),
+
+	icon: 'admin-users',
+
+	category: 'widgets',
+
+	keywords: [ __( 'author' ) ],
```

Comment:
You shouldn't need to define this keyword, since inserter search already uses the "title" field for search, and it's unlikely "Author" would be translated differently than "author".